<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>331</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Работа с файлами</string>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>150</width>
       <height>40</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>16</height>
      </size>
     </property>
     <property name="text">
      <string>Выбрать файл</string>
     </property>
    </widget>
    <widget class="QLabel" name="fileNameLabel">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>150</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Файл не выбран</string>
     </property>
    </widget>
    <widget class="QPushButton" name="b_graph">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>79</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Построить графики</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>0</y>
      <width>331</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Декомпозиция</string>
    </property>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>200</width>
       <height>40</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
     <property name="maxVisibleItems">
      <number>3</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhDigitsOnly</set>
     </property>
    </widget>
    <widget class="QPushButton" name="b_Decomp">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Построить</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>70</y>
       <width>150</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Epsilon</string>
     </property>
    </widget>
    <widget class="QPushButton" name="b_SaveFileTxt">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>161</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Сохранить в файл</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>0</y>
      <width>331</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Реконструкция</string>
    </property>
    <widget class="QPushButton" name="b_Reconst">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>191</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Сделать реконстукцию</string>
     </property>
    </widget>
    <widget class="QPushButton" name="b_SaveWav">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>161</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Сохранить .wav</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>200</y>
      <width>1300</width>
      <height>231</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Результаты</string>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
